cli/tests/pcluster/templates/test_imagebuilder_stack.py

```diff
@@ -689,6 +689,115 @@ def _test_resources(generated_resources, expected_resources):
             },
             {"Ref": "InstanceProfile"},
         ),
+        (
```

Comment:
new param missing from `imagebuilder_schema_dev.yaml`

Reply:
we can't put new param with custom InstanceRole. Shall I write a new template for new param unit test?